server/channels/api4/system.go

```diff
@@ -89,15 +89,27 @@ func generateSupportPacket(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// We support the existing API hence the logs are always included
+	// if nothing specified.
+	includeLogs := true
+	if r.FormValue("basic.server.logs") == "false" {
+		includeLogs = false
+	}
```

Comment:
Do we need to update the API documentation?

Reply:
Yes, but it's true by default so that we maintain backwards compatibility.

Reply:
Would you please create a follow-up ticket to document the API changes?